lib/fireblocks/jwt.go

```diff
@@ -12,25 +12,26 @@ import (
 	"github.com/google/uuid"
 )
 
+const expirationSecs = 29 // Must be less than 30sec.
+
 // genJWTToken generates a JWT token for the Fireblocks API.
 func (c Client) genJWTToken(uri string, data any) (string, error) {
```

Comment:
`genJWTToken` -> `genToken()` or even just `token()` because we return `string` and not a `*jtw.Token`, for example. The type of the token is an implementation detail that we can include in the comments.